backend/src/test/java/gov/cdc/usds/simplereport/service/AzureStorageQueueTestEventReportingServiceTest.java

```diff
@@ -45,6 +46,18 @@ void dispatches_supplied_event_to_queue_client() {
     verify(client, times(1)).sendMessage(argThat(matcherForTest(testEvent)));
   }
 
+  @Test
+  void throws_custom_test_event_serialization_failure_exception() {
+    var client = mock(QueueAsyncClient.class);
+    Mono<SendMessageResult> response = mock(Mono.class);
+    when(response.toFuture())
+        .thenReturn(CompletableFuture.completedFuture(new SendMessageResult()));
+    when(client.sendMessage(any(String.class))).thenReturn(response);
+
+    var sut = new AzureStorageQueueTestEventReportingService(new ObjectMapper(), client);
+    assertThrows(TestEventSerializationFailureException.class, () -> sut.report(new TestEvent()));
```

Comment:
nit: it may be worth naming this something like `invalidTestEventWithNoResults`. As it stands, it takes a bit of additional context to know that `new TestEvent()` by itself won't serialize correctly.

Reply:
(Sonar will probably complain at you for this and say you don't need a variable for a single-use piece of code, but I disagree lol)

Reply:
agree with Emma!

Reply:
Got it! Sonar is already mad at me for my lack of coverage so I'll change that so it's more readable! Thank you Emma and George!!